R/utils.R

```diff
@@ -0,0 +1,16 @@
+#' Get hub name for a given disease.
+#'
+#' Converts disease identifier to proper hub name format.
+#'
+#' @param disease Character. Disease identifier ("covid" or "rsv").
+#' @return Character. Hub name (e.g., "CovidHub", "RSVHub").
+#' @export
+get_hub_name <- function(disease) {
+  checkmate::assert_scalar(disease)
+  checkmate::assert_names(disease, subset.of = c("covid", "rsv"))
+
+  dplyr::case_when(
+    disease == "covid" ~ "CovidHub",
+    disease == "rsv" ~ "RSVHub"
+  )
```

Comment:
Style preference for explicit returns (and `invisible()` for functions that return `NULL`)
```suggestion
  return(dplyr::case_when(
    disease == "covid" ~ "CovidHub",
    disease == "rsv" ~ "RSVHub"
  ))
```